include/goofit/ThrustOverride.h

```diff
@@ -0,0 +1,108 @@
+#ifndef _ThrustOverride_h_
+#define _ThrustOverride_h_
+
+#include <thrust/execution_policy.h>
+#include <thrust/detail/config.h>
+#include <thrust/reduce.h>
+#include <thrust/detail/seq.h>
+#include <thrust/detail/temporary_array.h>
+#include <thrust/system/cuda/detail/bulk.h>
```

Comment:
These break the OMP build (for example, on Travis), since `detail/cuda` expects cuda to be available.